source/guide_neos.rst

```diff
@@ -0,0 +1,155 @@
+ Use it: php composer.phar
+
+ [isabell@stardust ~]$ mv composer.phar ~/bin/composer
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Since Neos uses the subdirectory Web/ as web root you should not install Neos in your `document root`_. Instead we install it next to that and then use a symlink to make it accessible.
+
+``cd`` to one level above your `document root`_, then use Composer to create a new project based on the Neos base distribution:
+
+.. note:: Composer will install all neccessary dependencies Neos needs to run. This can take some time.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/
+ [isabell@stardust isabell]$ composer create-project neos/neos-base-distribution Neos
+ Neos
+ Installing neos/neos-base-distribution (42.23.1)
+   - Installing neos/neos-base-distribution (42.23.1): Downloading (100%)
+ Created project in Neos
+ Loading composer repositories with package information
+ Updating dependencies (including require-dev)
+ [...]
+
+ [isabell@stardust isabell]$
+
+Remove your unused `document root`_ and create a new symbolic link to the Neos/Web directory:
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/
+ [isabell@stardust isabell]$ rm -rf html
+ [isabell@stardust isabell]$ ln -s /var/www/virtual/$USER/Neos/Web html
+ [isabell@stardust isabell]$
+
+Configuration
+============
+
+Point your browser to your website URL and append /setup (i.e. isabell.uber.space/setup) to use the web based installer.
+
+After setting some things up automatically, the setup will ask you for a setup password. You need to get this from a file in your Neos installation first: 
+
+::
+
+ [isabell@stardust isabell]$ cat /var/www/virtual/$USER/Neos/Data/SetupPassword.txt
+ The setup password is:
+
+ Z3pyq1Sp
+
+ After you successfully logged in, this file is automatically deleted for security reasons.
+ Make sure to save the setup password for later use.
+
+Step 1: After you logged in using your setup password Neos will check for supported image manipulation libraries. Fortunately on Uberspace 7 there are multiple libraries preinstalled. Neos will use "imagick" as standard. You can change that later manually.
+
+Step 2: To configure your database, you need to enter the following information:
+
+  * your preffered database driver. You can use the preselected ``MySQL/MariaDB via PDO``.
+  * your MySQL hostname, username and password: the hostname is ``localhost`` and you should know your MySQL credentials_ by now. If you don't, start reading again at the top.
+  * your Neos database name: we suggest you use an additional_ database. For example: isabell_neos
+
+.. warning:: Uberspace 7 databases are using the character set utf8mb4 by default. Unfortunately Neos is still using the utf8 character set and the utf8_unicode_ci collation by default. If you want to avoid mixed character sets you need to change this for your database manually. The setup tool won't do this for you. 
```

Comment:
I'm not shure about wether U7s default charachter set will impact usage if not changed. In my test installation I left it unchanged and there were no obvious errors because of utf8mb4.

Reply:
I _think_ this is only relevant when migrating an installation. I haven't used Neos though..

Reply:
@luto Any thoughts on this?

Reply:
neos has an tracking issue for this topic: https://github.com/neos/flow-development-collection/issues/1268 So I guess it is not yet fully compatible. The issue has an "Next Major" tag attached to it, so I guess we could wait for that release before merging this guide?

Reply:
I also looked into this yesterday and came to a similar conclusion regarding the issue and the pull requests. The next major release is due in the next weeks so waiting is the safest way we can go.

Reply:
All tables created by Neos will be utf8, no matter what the server default is. So it'll be fine either way. 👍

Reply:
Thanks for the info @kdambekalns!

Reply:
Thanks a lot @kdambekalns! So I removed the warning with 6c35065 and wrote an info that the installer can create the additional database for the user. So Neos can create the database with whatever charset's favored in the current release.